src/app/verify/mfa/index.njk

```diff
@@ -41,15 +41,16 @@
           </label>
           <span id="mfaCode-hint" class="govuk-hint">{{__('field_authentication_code_hint')}}</span>
           {{ m.error_message('mfaCode', errors, __) }}
-          <input id="mfaCode" autocomplete="off" class="govuk-input" maxlength="{{ mfaTokenLength }}" name="mfaCode" type="text" aria-describedby="mfaCode-hint mfaCode-error">
-          <p style="margin-top: 0.25em;" class="govuk-body-m">
-            <a href="/login/authenticate?resend=true">{{__('resend_code')}}</a>
-          </p>
+          <input id="mfaCode" autocomplete="off" class="govuk-input" maxlength="{{ mfaTokenLength }}" name="mfaCode" type="text" aria-describedby="mfaCode-hint{{ ' mfaCode-error' if errors}}">
```

Comment:
Moved the resend token button. I imagine a user with experience would tab and expect it to be the submit button only to find this one instead?